src/harness/infra/README.md

```diff
@@ -18,7 +18,7 @@ Shared Infrastructure components of Harness
 | postgresql.auth.existingSecret | string | `"postgres"` |  |
 | postgresql.commonLabels.app | string | `"postgres"` |  |
 | postgresql.fullnameOverride | string | `"postgres"` |  |
-| postgresql.image.tag | string | `"14.7.0-debian-11-r28"` |  |
```

Comment:
we planning to use this in July end release ?

Reply:
Yes, here is the corresponding PR for release branch:  https://github.com/harness/helm-charts/pull/357